### PR TITLE
Set: Landmark roles for screen reader

### PR DESCRIPTION
**Pull request by andygout**
cc @ironsidevsquincy 

Top Stories section (along with fastFT) now listed as 'main', with subsequent sections as 'complementary'.

fastFT sidebar exists as an aside (default ARIA role value is 'complementary') within the 'main' section, which seems to be acceptable providing that the aside relates to the 'main', which in this context feels it does: http://stackoverflow.com/questions/17549780/can-you-use-an-aside-element-inside-a-section-role-main-element

[W3C - The Roles Model: `main`](https://www.w3.org/TR/wai-aria/roles#main)
[W3C - The Roles Model: `complementary`](https://www.w3.org/TR/wai-aria/roles#complementary)

### CODE REVIEW

File: views/front-page.html [hunk 1]
```diff
@@ -50,8 +50,10 @@
 	{{#each sections}}
 		<div
 			id="{{id}}"
+			{{#if role}}role="{{role}}"{{/if}}
 			{{#if trackScrollEvent}} class="js-track-scroll-event"{{/if}}
-			data-o-grid-colspan="{{colspan size}}">{{#unless isClient}}{{{reactRenderToString ../Section spread=@this}}}{{/unless}}</div>
+			data-o-grid-colspan="{{colspan size}}">{{#unless isClient}}{{{reactRenderToString ../Section spread=@this}}}{{/unless}}
+		</div>
```
Comment (ironsidevsquincy) [inline views/front-page.html]: think the closing div needs to be on the same line, or else react complains about empty spaces

**Comment by ironsidevsquincy** [comment]
Is the fast ft block related to the top stories?

**Comment by andygout** [comment]
Following discussion, will contain all sections with `role="main"` and address any elements contained therein that default to `role="complementary"` (disabling that role value).

**Comment by andygout** [comment]
Given I have changed the `aside` elements to `div`s, I have also amended the fastFT section's associated values from 'aside' to 'sidebar' given it is referred to as such (or 'sidebarComponent').

**Comment by ironsidevsquincy** [comment]
:+1:

**Pull request merged**